components/list/list-item-mixin.js

```diff
@@ -38,9 +38,12 @@ function addTabListener() {
 	});
 }
 
+const listItemInteractiveFlag = window.D2L?.LP?.Web?.UI?.Flags.Flag('GAUD-7495-list-interactive-content', true) ?? true;
```

Comment:
We have [a helper](https://github.com/BrightspaceUI/core/blob/main/helpers/flags.js) for this now!